unit_test/main_options_test.py

```diff
@@ -70,6 +70,11 @@ def test_build_selector(platform, intercepted_build_args, monkeypatch):
     ('i686', 'manylinux2010', 'quay.io/pypa/manylinux2010_i686'),
     ('i686', 'manylinux2014', 'quay.io/pypa/manylinux2014_i686'),
     ('i686', 'custom_image', 'custom_image'),
+    ('pypy_x86_64', None, 'pypywheels/manylinux2010-pypy_x86_64'),
+    ('pypy_x86_64', 'manylinux1', 'manylinux1'),  # Does not exist
+    ('pypy_x86_64', 'manylinux2010', 'pypywheels/manylinux2010-pypy_x86_64'),
+    ('pypy_x86_64', 'manylinux2014', 'manylinux2014'),  # Does not exist (yet)
+    ('pypy_x86_64', 'custom_image', 'custom_image'),
```

Comment:
also here upgrade expected image to `yannickjadoul/manylinux2010-pypy_x86_64:7.3.0`

Reply:
I know. It was already fixed, but I still needed to wait for the image to be pushed.